root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -87,6 +90,11 @@ if [[ -n $database ]]; then
 
     if  ! databaseTest ; then
         OCC config:system:set dbhost --value="localhost:/run/rh-mariadb105-mariadb/nextcloud-mysql.sock" --type="string"
+        if [[ $? -gt 0 ]]; then
```

Comment:
I do not understand here on[ line 12 ](https://github.com/NethServer/nethserver-nextcloud/blob/2393384fb82abf866cb5a8380c318811d6ae902c/root/etc/e-smith/events/actions/nethserver-nextcloud-conf#L12)

We disable maintenance mode and enable accessibility with the function `exitOnError` I believe we do not need it, only exitOnError if we cannot configure the mysql socket

Reply:
You're right.
Fixed